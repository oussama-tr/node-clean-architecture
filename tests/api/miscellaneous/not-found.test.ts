import { makeClient } from '../client';

describe('GET /does-not-exist', () => {
  test('responds with a 404 status code', async () => {
    // Arrange
    const { client } = await makeClient();

    // Act
    const response = await client.get('/does-not-exist');

    // Assert
    expect(response.status).toEqual(404);
    expect(response.body).toMatchSnapshot();
  });
});
