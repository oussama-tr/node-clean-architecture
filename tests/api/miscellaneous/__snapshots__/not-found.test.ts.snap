// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /does-not-exist responds with a 404 status code 1`] = `
{
  "error": "Not Found",
  "message": "Route GET:/does-not-exist not found",
  "statusCode": 404,
}
`;
